Makefile

```diff
@@ -33,69 +38,77 @@ help:
 	@echo "  dropdb           Completely remove the postgres container and its data"
 	@echo "  dumpdb           Create a postgres database dump with timestamp used as file name"
 	@echo "  loaddb           Load a database dump into postgres, file name in DB_DUMP_FILE"
-	@echo "  build-frontend   Builds the frontend static files"
-	@echo "  build-frontend-w Watches the frontend static files and builds on change"
+	@echo "  build-tagadmin   Builds the tag_admin frontend static files"
+	@echo "  build-tagadmin-w Watches the tag_admin frontend static files and builds on change"
 	@echo "  sync-projects    Runs the synchronization task on all projects"
 	@echo "  requirements     Compiles all requirements files with pip-compile\n"
 
-.docker-build:
-	make build
+.frontend-build:
+	make build-frontend
+.server-build:
+	make build-server
 
-build:
-	cp ./docker/config/webapp.env.template ./docker/config/webapp.env
-	sed -i -e 's/#SITE_URL#/$(subst /,\/,${SITE_URL})/g' ./docker/config/webapp.env
+build: build-frontend build-server
+build-frontend: server-env
+	"${DC}" build frontend
+	touch .frontend-build
+build-server: server-env
+	"${DC}" build --build-arg USER_ID=$(USER_ID) --build-arg GROUP_ID=$(GROUP_ID) server
+	touch .server-build
 
-	"${DC}" build --build-arg USER_ID=$(USER_ID) --build-arg GROUP_ID=$(GROUP_ID) webapp
+server-env:
+	cp ./docker/config/server.env.template ./docker/config/server.env
+	sed -i -e 's/#FRONTEND_URL#/$(subst /,\/,${FRONTEND_URL})/g;s/#SITE_URL#/$(subst /,\/,${SITE_URL})/g' ./docker/config/server.env
 
-	touch .docker-build
+setup: .server-build
+	"${DC}" run server //app/docker/server_setup.sh
 
-setup: .docker-build
-	"${DC}" run webapp //app/docker/set_up_webapp.sh
-
-run: .docker-build
-	"${DC}" run --rm --service-ports webapp
+run: .frontend-build .server-build
+	"${DC}" up
 
 clean:
-	rm .docker-build
+	rm -f .docker-build .frontend-build .server-build
+
+test: test-server test-frontend
 
-test:
-	"${DC}" run --rm webapp //app/docker/run_tests.sh
+test-server:
+	"${DC}" run --rm server //app/docker/server_tests.sh
```

Comment:
I think `test-server` should only run unit tests (i.e. pytest): https://github.com/mozilla/pontoon/pull/1996#discussion_r706076167.

Reply:
Done.